### ISSUE TITLE
Incorrect display of the title page in web browser extensions

### ISSUE THREAD

**Issue by Ubuntu-2020** [kiwix/kiwix-desktop#482]
The title pages of some ZIM files are not displayed correctly in web browser extensions. Here are some examples:
**wikipedia_ru_history_maxi_2020-05.zim**
![wikipedia_ru_history_maxi_2020-05 zim](https://user-images.githubusercontent.com/65112331/86588591-a9058e00-bf94-11ea-8f49-dc3e9ecb8950.png)

**wikipedia_ru_football_maxi_2020-04.zim**
![wikipedia_ru_football_maxi_2020-04 zim](https://user-images.githubusercontent.com/65112331/86588583-a4d97080-bf94-11ea-85d0-1b52c1af6737.png)

**wikipedia_ru_movies_maxi_2020-05.zim**
![Kiwix 2 7 0 extension for web browser Vivaldi-ru_movies_maxi_2020-05](https://user-images.githubusercontent.com/65112331/86588499-7e1b3a00-bf94-11ea-9bdb-70b387a97c7a.png)

In other words, you can't scroll the main page with pictures down in the browser! There is a huge black area at the bottom.
Images were taken in the Mozilla FireFox web browser.

**Comment by kelson42** [comment]
@Ubuntu-2020 How it should looks like?

**Comment by kelson42** [comment]
@Ubuntu-2020 Please open Kiwix-JS bugs at https://github.com/kiwix/kiwix-js

**Comment by kelson42** [comment]
@Ubuntu-2020 please amke a proposal here, because to me, there is not bug.

**Comment by Ubuntu-2020** [comment]
>@kelson42 please make a proposal here, because to me, there is not bug.

If this is not a bug, what is it?
**wikipedia_ru_football_maxi_2020-04.zim** - **MozillaFireFox** extension **Kiwix 2.8.0**.
![wikipedia_ru_football_maxi_2020-04](https://user-images.githubusercontent.com/65112331/87245572-fd09ea00-c44e-11ea-9494-2fbfd61bfeaa.png)

Only the top row of images is visible. And from below, a huge black space. This way it is impossible to scroll the mouse and go down!

**Comment by kelson42** [comment]
> Only the top row of images is visible

I better understand the problem now, it seems we have lost the ability to scroll on this page! This seems to be a regression in Kiwix-JS, but not sure. Moving the ticket.
